tests/trainer/test_trainer.py

```diff
@@ -1250,6 +1250,44 @@ def test_accumulate_time_across_ranks(
         assert num_tokens_accum == num_tokens * 2
         assert batch_time_accum == datetime.timedelta(seconds=0.1 * (1 + 0))
 
+    @pytest.mark.gpu
```

Comment:
```suggestion
```
since the model is small, test shouldn't need a GPU

Reply:
Can it still correctly run with a world size of 2 without GPU?

Reply:
Yep! We have world size 2 tests for CPU models

Reply:
Nice, have done that :)